app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
Phân biệt cho chị `present?` và `nil?` khác nhau thế nào nhé

Reply:
prisent là check not nil, not empty, not false. Còn nil? là check object là nil hay không, nếu nil trả về true, ngược lại trả về false

Reply:
ở đây dùng present? để khỏi dùng nil? + nghịch đảo kết quả